### PR TITLE
Add sensor ambient_station

### PR DESCRIPTION
**Pull request by tmd224**
**Description:**
Adding documentation to support new Ambient Weather Station sensor component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18551

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.ambient_station.markdown [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+layout: page
+title: "Ambient Weather Station Sensor"
+description: "How to integrate Ambient Weather station within Home Assistant."
+date: 2018-11-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Weather
+featured: false
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]: This is the default.

File: source/_components/sensor.ambient_station.markdown [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ha_release: "0.83"
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]: 0.85

File: source/_components/sensor.ambient_station.markdown [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+ha_iot_class: "Cloud Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown:12]: Is there no logo?

File: source/_components/sensor.ambient_station.markdown [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+The `Ambient Weather Station` platform uses the [Ambient Weather](https://ambientweather.net)  
+web API to retrieve weather data from your personal weather station (PWS).
+
+You need to register your compatible PWS using its MAC address and then request both an API key and an Application key.  The key requests can be found
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]: Are there two spaces between `key.` and `The key`?

File: source/_components/sensor.ambient_station.markdown [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  - platform: ambient_station
+    api_key: YOUR_API_KEY
+    app_key: YOUR_APP_KEY
+    monitored_conditions:
+      - tempf
+      - humidity
+      - windspeedmph
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]: Use the new configuration tag.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

File: source/_components/sensor.ambient_station.markdown [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  - **baromrelin**:
+  - **baromabsin**:
+  - **uv**:
+  - **solarradiation**: 
+  - **co2**:
+  - **lastRain**:
+  - **dewPoint**: 
+  - **feelsLike**:
+  
+- **scan_interval** (*Optional*): Minimum time interval between updates. Default is 5 minutes. Supported formats:
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]: Remove this. All auto polling platforms support this.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/sensor.ambient_station.markdown (hunk 3 above)
Comment (tmd224) [reply source/_components/sensor.ambient_station.markdown:12]: Yes, added logo

File: source/_components/sensor.ambient_station.markdown (hunk 1 above)
Comment (tmd224) [reply source/_components/sensor.ambient_station.markdown]: Removed

File: source/_components/sensor.ambient_station.markdown [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+layout: page
+title: "Ambient Weather Station Sensor"
+description: "How to integrate Ambient Weather station within Home Assistant."
+date: 2018-11-15 08:00
+sidebar: true
+comments: false
+logo: ambient_weather.png
+ha_release: "0.85"
+ha_iot_class: "Cloud Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown:12]: We should keep the weather category.

File: source/_components/sensor.ambient_station.markdown [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+      - windspeedmph
+```
+
+{% configuration %}
+api_key:
+  description: The API key to access the service.
+  required: true
+  type: string
+app_key:
+  description: The Application key to access the service
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]:
```suggestion
  description: The Application key to access the service.
```

File: source/_components/sensor.ambient_station.markdown [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+  required: true
+  type: string
+monitored_conditions:
+  description: Weather conditions to display in the frontend.
```
Comment (MartinHjelmare) [inline source/_components/sensor.ambient_station.markdown]:
```suggestion
  description: Weather conditions to track.
```

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Comments by @MartinHjelmare are addressed. I've reviewed it as well and LGTM.
Thanks, @tmd224!

**Pull request merged**